internal/runners/packages/packages.go

```diff
@@ -53,22 +52,23 @@ func executePackageOperation(prime primeable, packageName, packageVersion string
 	langName := "undetermined"
 
 	out := prime.Output()
-	var pg *output.DotProgress
+	var pg *output.Spinner
 	defer func() {
-		if pg != nil && !pg.Stopped() {
+		if pg != nil {
 			pg.Stop(locale.T("progress_fail"))
 		}
 	}()
 
 	var err error
 	pj := prime.Project()
 	if pj == nil {
-		pg = output.NewDotProgress(out, locale.Tl("progress_project", "", packageName), 10*time.Second)
+		pg = output.StartSpinner(out, locale.Tl("progress_project", "", packageName), constants.TerminalAnimationInterval)
 		pj, err = initializeProject()
 		if err != nil {
 			return locale.WrapError(err, "err_package_get_project", "Could not get project from path")
 		}
 		pg.Stop(locale.T("progress_success"))
+		pg = nil
```

Comment:
Might be worth adding a comment explaining why this is necessary (it's so we don't redundantly call pg.Stop())